### ISSUE TITLE
TPOTClassifier RuntimeError: There was an error in the TPOT optimization process. 

### ISSUE THREAD

**Issue by RazHoshia** [EpistasisLab/tpot#1156]
I got RuntimeError when trying to run a simple binary classification problem on both TPOT cuML and default configuration.
I tried changing the target's type (Tried label encoded int, string (object), and boolean) to try to resolve the error `because data for a regression problem was provided to the TPOTClassifier object`. I have also tried to change the population size and the number of generations.

## Code

```
X_test = test.drop([TARGET_COL], axis=1)
# dataset is already label encoded after Pipeline
y_train = train[TARGET_COL]
X_train = train.drop([TARGET_COL], axis=1)

X_train.fillna(numpy.finfo(numpy.float64).min, inplace=True)  # else cause TPOT error

pipeline_optimizer = TPOTClassifier(generations=100, population_size=100, cv=2,
                                    random_state=1234, verbosity=3,
                                    early_stop=5,
                                    # config_dict="TPOT cuML"
                                    max_time_mins=100000,
                                    scoring='f1'
                                    )

pipeline_optimizer.fit(X_train, y_train.array)

```

## Expected result

trained pipeline_optimizer object.

## Current result

`RuntimeError: There was an error in the TPOT optimization process. This could be because the data was not formatted properly, or because data for a regression problem was provided to the TPOTClassifier object. Please make sure you passed the data to TPOT correctly. If you enabled PyTorch estimators, please check the data requirements in the online documentation: https://epistasislab.github.io/tpot/using/`

**Comment by weixuanfu** [comment]
Please check #1148 and #1153.

**Comment by weixuanfu** [comment]
This issue should be fixed in recent release (v0.11.7) of TPOT. Please update TPOT and feel free to reopen this issue if it still exists.
